### PR TITLE
Documentation added

### PR DESCRIPTION
**Pull request by MisterIXI**

### CODE REVIEW

File: connection_steps.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    1. Switch it on
+    1. Wait for the robot to boot and login
+    1. Unplug the HDMI cable
+    1. Plug the USB cable back in
+1. Make sure configurations in the .bashrc are right
+    1. In the Ubuntu master
+        1. `export TURTLEBOT3_MODEL=waffle`
+        1. `export ROS_MASTER_URI=http://[Ubuntu_IP]:11311`
+        1. `export ROS_HOSTNAME=[Ubuntu_IP]`
+    1. In the Robot
```
Comment (v0rl0ck) [inline connection_steps.md]: typo in "Robot", it is a small 'r'

Comment (v0rl0ck) [inline connection_steps.md]: typo in "Robot", it is a small 'r'

File: learnings.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# X11 forwarding
+To forward RVIZ and other windows from the Ubuntu Master to the Host Machine.
+If everything is set up correctly you have to do the following steps:
+1. On the Ubuntu Master, run`export DISPLAY=[Host Machine]:0`
```
Comment (v0rl0ck) [inline learnings.md]: There is a space missing between "run" and the command

**Review by v0rl0ck — CHANGES_REQUESTED** [review]

File: learnings.md (hunk 2 above)
Comment (MisterIXI) [reply learnings.md]: fixed

File: connection_steps.md (hunk 1 above)
Comment (MisterIXI) [reply connection_steps.md]: fixed

**Review by v0rl0ck — APPROVED** [review]

**Pull request merged**